public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -295,58 +280,16 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
       })),
     };
 
-    if (options.makeReplacements) {
-      requestParams.queries.forEach((query: CloudWatchLogsRequest) => {
-        const fieldsToReplace: Array<
-          keyof (GetLogEventsRequest & StartQueryRequest & DescribeLogGroupsRequest & GetLogGroupFieldsRequest)
-        > = ['queryString', 'logGroupNames', 'logGroupName', 'logGroupNamePrefix'];
-
-        // eslint-ignore-next-line
-        const anyQuery: any = query;
-        for (const fieldName of fieldsToReplace) {
-          if (query.hasOwnProperty(fieldName)) {
-            if (Array.isArray(anyQuery[fieldName])) {
-              anyQuery[fieldName] = anyQuery[fieldName].flatMap((val: string) => {
-                if (fieldName === 'logGroupNames') {
-                  return this.expandVariableToArray(val, options.scopedVars || {});
-                }
-                return this.replaceVariableAndDisplayWarningIfMulti(val, options.scopedVars, true, fieldName);
-              });
-            } else {
-              anyQuery[fieldName] = this.replaceVariableAndDisplayWarningIfMulti(
-                anyQuery[fieldName],
-                options.scopedVars,
-                true,
-                fieldName
-              );
-            }
-          }
-        }
-
-        if (anyQuery.region) {
-          anyQuery.region = this.replaceVariableAndDisplayWarningIfMulti(
-            anyQuery.region,
-            options.scopedVars,
-            true,
-            'region'
-          );
-          anyQuery.region = this.getActualRegion(anyQuery.region);
-        }
-      });
-    }
-
     const resultsToDataFrames = (
       val:
         | { data: BackendDataSourceResponse | undefined }
         | FetchResponse<BackendDataSourceResponse | undefined>
         | DataQueryError
     ): DataFrame[] => toDataQueryResponse(val).data || [];
     let headers = {};
-    if (options.skipCache) {
```

Comment:
Now none of the calling methods wants to skip cache, so I removed this argument.

Reply:
Why did they want to skip the cache before and why do they not now?

Reply:
Similar to the long running query feature in Athena and Redshift, CW logs polls the grafana backend (and the logs apis) to check if there are results to fetch. Unlike the Athena and Redshift apis, the logs api can also return partial responses. This is neat because even though the query may need to scan petabytes of data which can take minutes, you may be able to see some result on the logs panel after only a few seconds. However, if we cache a partial response, it will mess up query caching pretty badly. For that reason, none of query data requests (StartQuery, GetQueryResult, StopQuery) should be cached. Some of the requests that have now been moved to the resource request flow (through api.ts) used to set the `skipCache` parameter to false, but now that they're not using the `makeLogActionRequest` anymore, we can safely remove the argument from this function since all the calling methods always wants the cache to be bypassed. 

Let's talk more about this in the mob session!